### PR TITLE
Cview-issuer for cert-manager

### PR DESCRIPTION
**Pull request by djkormo**
New issuer has been created for cert-manager. It uses cview as dedicated CA. The website of this produkt product https://www.secure-ly.com/c-view-professional-certificates-lifecycle-management/ . Issuer is not open-sourced.

### CODE REVIEW

**Comment by Dror1966** [comment]
Hi, Can you prioritize this issue?

**Comment by djkormo** [comment]
@jakexks  Please help me with this npm check. I do not understand why is is failed.

**Comment by djkormo** [comment]
The spelling has been corrected.

**Review by SgtCoDFish — CHANGES_REQUESTED** [review]
Hey, thanks for contributing!

It looks like the [deploy preview](https://deploy-preview-1541--cert-manager.netlify.app/docs/configuration/issuers/) is showing a broken link for this. Looks like the `ca:cview-issuer` link hasn't been added.

Could you fix that? I'd be happy to merge then!

Screenshot 2024-09-23 at 15 01 43

File: content/docs/configuration/issuers.md
Comment (inteon) [inline content/docs/configuration/issuers.md:18]: Do you have a releases page? We have to make sure the issuer has had a release in the last 12 months.

Also, can you confirm the following (does not require a proof):
- These issuers are known to support and honor [approval](https://cert-manager.io/docs/concepts/certificaterequest/#approval).

Comment (djkormo) [reply content/docs/configuration/issuers.md]: There is a release page for helm chart. But the release of issuer image is in a private repo.
Images for issuer are visible on docker hub . https://hub.docker.com/r/devsecurely/cview-issuer/tags

**Review by SgtCoDFish — DISMISSED** [review]
Thanks for updating the link, it looks correct now!

There's one last thing based on [inteon's comment](https://github.com/cert-manager/website/pull/1541#discussion_r1771550161) - can you confirm that your issuer honors approval?

That is, it will not issue for a cert request which is not marked "Approved"?

If your issuer is based on `issuer-lib` then it should handle approval. If it's more custom it might not, so we need to check!

**Review by inteon — COMMENTED** [review]
/approve
/lgtm
/hold We can unhold once you confirm it does handle approval.

**Comment by djkormo** [comment]
Code was based on the beginning from this repo 
https://github.com/cert-manager/sample-external-issuer/tree/main

It does handle approval.

**Pull request merged**
